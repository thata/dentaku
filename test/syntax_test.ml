let _ =
  print_int 4649;
  print_newline ()

let _ =
  let n = Eval.f (Syntax.Num(5963)) in
  print_int n;
  print_newline ()