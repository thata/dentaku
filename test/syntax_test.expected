4649
5963
